php/ng/dba.sls

```diff
@@ -0,0 +1,2 @@
+{% set state = 'adodb' %}
```

Comment:
I think there's a typo here: it should be dba

Reply:
You are right, I have fixed it. Although I think #177 is better, supports other distributions